chore(#2199): update steps on how to add new keys and LLM use (#2210)
medic/cht-docs: content/en/community/contributing/translations.md

- First check if an appropriate key already exists in [`messages-en.properties`](https://github.com/medic/cht-core/blob/master/api/resources/translations/messages-en.properties). Using existing keys when possible reduces the effort required to translate the app into a new language.
- Create a unique key to identify the translation string. The key should accurately describe the string's purpose/usage.
- Add the key to each of the files contained in `api/resources/translations`.
- Add translations for your key and strings in English and any other languages you are proficient in. For languages you're not proficient in, you may use translation tools (e.g. LLMs like Claude, Gemini, or ChatGPT; translation services; or help from fluent speakers) to draft them.
- Translations are required for all the languages marked as "Supported" in the table above.

> - For any translations you didn't author as a fluent speaker, make a post in the [Translation category](https://forum.communityhealthtoolkit.org/c/product/translations/35) on the CHT Community Forum requesting validation from native speakers. Include a link to the PR so that corrections can be suggested easily.

Reviewer: ```suggestion
- For any translations you didn't author as a fluent speaker, make a post in the [Translation category](https://forum.communityhealthtoolkit.org/c/product/translations/35) on the CHT Community Forum requesting validation from fluent speakers. Include a link to the PR so that corrections can be suggested easily.
```

- First check if an appropriate key already exists in [`messages-en.properties`](https://github.com/medic/cht-core/blob/master/api/resources/translations/messages-en.properties). Using existing keys when possible reduces the effort required to translate the app into a new language.
- Create a unique key to identify the translation string. The key should accurately describe the string's purpose/usage.
- Add the key to each of the files contained in `api/resources/translations`.

> - Add translations for your key and strings in English and any other languages you are proficient in. For languages you're not proficient in, use an LLM like Claude, Gemini or ChatGPT to generate the translations.

Reviewer: ```suggestion
- Add translations for your key and strings in English and any other languages you are proficient in. For languages you're not proficient in, you may use translation tools (e.g. LLMs like Claude, Gemini, or ChatGPT; translation services; or help from fluent speakers) to draft them.
```

Reviewer: My worry is that this pulls us back into AI-specific framing: someone using Google Translate, a translation service, or asking a bilingual friend or family member would read this and be unsure whether the same rules apply. In practice, we can't verify what tool was used, and it doesn't really matter until peer review happens. Something like "you may use translation tools (LLMs, machine translation, etc.) or ask a fluent speaker" keeps it inclusive of however contributors actually work.

> - Translations are required for all the languages above marked as "Supported".

Reviewer: ```suggestion
- Translations are required for all the languages marked as "Supported" in the table above.
```

Reviewer: nitpick for extra clarity

> - Once all the required launguages are added and [translation linting](/community/contributing/translations/#linting-translations) passes, the CHT Core PR can be merged.

Reviewer: ```suggestion
- For any translations you didn't author as a fluent speaker, make a post in the [Translation category](https://forum.communityhealthtoolkit.org/c/product/translations/35) on the CHT Community Forum requesting validation from native speakers. Include a link to the PR so that corrections can be suggested easily.
- Once all the required languages are added, any non-fluent translations have been validated by community members and [translation linting](/community/contributing/translations/#linting-translations) passes, the CHT Core PR can be merged.
```

> If an LLM translated strings where used, make a forum post in the [Translation category](https://forum.communityhealthtoolkit.org/c/product/translations/35) requesting validation. This way any native speakers from the community can ensure LLM translations are accurate.  Include a link to the PR so that corrections can easily be requested.

Reviewer: ```suggestion
```

Reviewer: Bumped this one to one bullet point above.
